inst/js/tableClick.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Listens for clicks on a table and highlights the site selector
+ *
+ * @param {HTMLElement} tableId - The unique element id for the table's parent div for the script to listen
+ * @returns {undefined}
+ */
+
+ const tableClick = function (tableId) {
+
+  document.getElementById(tableId).addEventListener('click', function(event) {
+    var target = event.target;
+    while (target && target.tagName !== 'TR') {
+      target = target.parentElement;
+    }
+
+    if (target && target.tagName === 'TR') {
+      var firstColumnValue = target.querySelector('td:first-child').textContent.trim().replace(/ \(.*\)$/, '');
+      console.log(firstColumnValue);
```

Comment:
```suggestion
```
Remove this. I assume it was for debugging, but now it clutters things up when I went in to try to make it work on tab switch 🙃